prow/cluster/jobs/rh-ibm-synergy/multicloud-operators-cluster-controller/rh-ibm-synergy.multicloud-operators-cluster-controller.master.yaml

```diff
@@ -25,7 +25,7 @@ presubmits:
       - command:
         - make
         - build
-        image: quay.io/multicloudlab/build-tools:v20191217-d33a10d
+        image: quay.io/multicloudlab/multicloudlab-builder:v20191217-d33a10d
```

Comment:
What is the reason to change the image name? 

/cc @morvencao

Reply:
@morvencao mentio that since my build step also build images due to operator-sdk it need the docker tools that's not in the build-tools image